client/src/App.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I think you should add some normalize stuff
at least
```css
box-sizing: border-box;
```

Reply:
I added `Reset CSS` and `box-sizing` to the root `App.css` file